include/RAJA/policy/sycl/reduce.hpp

```diff
@@ -104,12 +104,19 @@ struct Reduce_Data
    *
    *  allocates data on the host and device and initializes values to default
    */
-  Reduce_Data(T /*defaultValue*/, T identityValue, Offload_Info &info)
-      : value(identityValue),
-        device{reinterpret_cast<T *>(
-            cl::sycl::malloc_device(sycl::MaxNumTeams * sizeof(T),*(::RAJA::sycl::detail::getQueue())))},
-        host{new T[sycl::MaxNumTeams]}
+  Reduce_Data(T initValue, T identityValue, Offload_Info &info)
+      : value(initValue)
   {
+    cl::sycl::queue* q = ::RAJA::sycl::detail::getQueue();
+
+    if(!q) {
+      camp::resources::Resource res = camp::resources::Sycl();
+      q = res.get<camp::resources::Sycl>().get_queue();
+    } 
+
```

Comment:
We use an assigned sycl resource object here, and IIRC we don't need a "default" sycl resource like with CUDA / HIP because the Sycl backend handles the queue instantiation, right?

Reply:
We use one because Sycl does not provide a default context so a newly constructed queue will be in a different context.  Intel provides an extension for a default context.